study.md

```diff
@@ -50,15 +50,15 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+The controller layer of Rails receives information from the router that tell it which action to retrieve or save information from the model. The controller in addition uses the data from the model and sends it to the view to create HTML output. The controller is thus a middle-man for the view and the model as well as also receiving information from the router.
```

Comment:
We won't be using Rails's views so our controllers will only be rendering responses (usually JSON, but other things are possible too) to the client.